src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -244,6 +254,9 @@ export default function NurseryWithdrawals(props: NurseryWithdrawalsProps): JSX.
       case 'purposes': {
         return strings.PURPOSE;
       }
+      case 'withdrawnDates': {
+        return strings.DATE;
```

Comment:
strings.WITHDRAWN_DATE? i know it's internal to code but still?

Reply:
Oh, yes. This name appears on the pill, so it's not internal

Reply:
Thanks for fixing 🎉 everything looks good now.
Btw, quick question for visual update follow up, should we use that white background with 32px border radius on each of those filter sections? or leave for later until Rachel has new designs?

Reply:
I'm not sure which filter sections you are referring to?

Reply:
each category of filters